site/content/docs/03-template-syntax.md

```diff
@@ -933,7 +933,7 @@ Actions are functions that are called when an element is created. They can retur
 
 An action can have a parameter. If the returned value has an `update` method, it will be called whenever that parameter changes, immediately after Svelte has applied updates to the markup.
 
-> Don't worry about the fact that we're redeclaring the `foo` function for every component instance — Svelte will hoist any functions that don't depend on local state out of the component definition.
+> Don't worry about the fact that we're redeclaring the `foo` function for every component instance — Svelte will hoist any functions that don't depend on the local state out of the component definition.
```

Comment:
I think this sounds better without the "the".

Reply:
```suggestion
> Don't worry about the fact that we're redeclaring the `foo` function for every component instance — Svelte will hoist any functions that don't depend on local state out of the component definition.
```